### ISSUE TITLE
Adjust logo spacing, font sizes and alignment

### ISSUE THREAD

**Issue by ghing** [openelections/openelections.github.io#10]
Use dimensions from https://drive.google.com/file/d/0B0i_cKL1FSAfU2hucXljb0p6OHM/edit?usp=sharing

**Comment by jotasprout** [comment]
If this issue is still current, I'd love to take it on.

**Comment by dwillis** [comment]
@jotasprout sure!

**Comment by jotasprout** [comment]
Starting on this but have some questions about the markings in the PDF. The red text on the right: Are those the specs for what you want or what they are currently? Some of the markings on the page itself (in the PDF) are already like that and some are pretty close -- but those are what you want, correct (they're not "current" measurements)? Do you really want the map of the USA to only be 185 px? It is currently ~766 px.

**Comment by dwillis** [comment]
Correct, those are what we want. I think the map should be ~785px, not 185px.

**Comment by jotasprout** [comment]
Cool. Got it. Thanks.
